### ISSUE TITLE
Maps with packed content cause CMaterial::PrecacheVars errors + lag after switching off them.

### ISSUE THREAD

**Issue by wrefgtzweve** [Facepunch/garrysmod-issues#5870]
### Details
Loading into any map after playing on a map with packed content will cause CMaterial::PrecacheVars errors. I'm not exactly sure why or how long it takes before it happens on the next map but it seems to happen roughly at the same time for all clients on the server so i assume something is triggering it.
Console will effectively be flooded with dozens of these lines:
```
CMaterial::PrecacheVars: error loading vmt file for models/props_c17/door01g_skin2
CMaterial::PrecacheVars: error loading vmt file for models/props_c17/door01g_skin3
CMaterial::PrecacheVars: error loading vmt file for models/props_c17/door01g_skin4
CMaterial::PrecacheVars: error loading vmt file for models/props_c17/door01g_skin5
CMaterial::PrecacheVars: error loading vmt file for models/props_c17/door01g_skin6
CMaterial::PrecacheVars: error loading vmt file for models/props_c17/door01g_skin7
```
This causes a massive lag spike for players.
This is extra annoying for gamemodes like TTT which switch map rather frequently.

### Steps to reproduce

The main issue is that i don't know exactly how to trigger the lag pike, it seems to be rather random.
1. Play on a map with custom content packed into the map
2. Load into any other map
3. Wait a certain amount of time and experience the lag spike

**Comment by Noahbg** [comment]
I'm also experiencing the same issue: random lag spikes after that console spew.

**Comment by StrawWagen** [comment]
Same here.

**Comment by robotboy655** [comment]
Can you please provide a link to a map that is known to cause the issue?

Not sure what I can do with "Wait a certain amount" at any rate.

**Comment by wrefgtzweve** [comment]
Switching from <https://steamcommunity.com/sharedfiles/filedetails/?id=2010286798> or <https://steamcommunity.com/sharedfiles/filedetails/?id=2925489145> to `gm_construct` will trigger it on spawning in 
![image](https://github.com/Facepunch/garrysmod-issues/assets/69946827/0073c246-0e6e-403d-8996-a0c3188698eb)
Not exactly the same as the issue players have as that happens even after spawning in, but that can happen multiple times so im assuming that its the same issue.

**Comment by brandonsturgeon** [comment]
Today I was seeing this on [`rp_southside`](https://steamcommunity.com/workshop/filedetails/?id=2010286798) but I'm not sure it was a map issue.

It happened to many players at the same time, and it was inconsistent. To me, it felt like it was the result of something a player was doing.
```
Attemped to precache unknown particle system "unusual_northstar_smoke_red"!
CMaterial::PrecacheVars: error loading vmt file for models/ttt/deerstalker/deerstalker
CMaterial::PrecacheVars: error loading vmt file for models/natalya/toilet/toilet
CMaterial::PrecacheVars: error loading vmt file for vgui/ttt/sample_scan
CMaterial::PrecacheVars: error loading vmt file for vgui/ttt/tbut_hand_line
CMaterial::PrecacheVars: error loading vmt file for vgui/ttt/tbut_hand_filled
CMaterial::PrecacheVars: error loading vmt file for vgui/ttt/score_logo
CMaterial::PrecacheVars: error loading vmt file for sprites/scope
```

**Comment by wrefgtzweve** [comment]
I just found a way to reliably trigger it. 
```lua
PrecacheParticleSystem( "invalidparticle" )
```
![image](https://github.com/Facepunch/garrysmod-issues/assets/69946827/fbc5f6a6-6945-4846-87f8-6df7c74c7d0e)

**Comment by brandonsturgeon** [comment]
Yeah it seems like if you `PrecacheParticleSystem` an invalid particle, it will print warnings about stuff from the previous map.

You can probably reproduce:
- Join some map with custom models/mats/etc
- Go back to flatgrass
- `PrecacheParticleSystem( "invalid" )`

**Comment by wrefgtzweve** [comment]
Seems to help a lot yeah!
I did notice that a 3 still showed up but there wasn't any lag.
```
Attemped to precache unknown particle system "qudtchfnrs"!
CMaterial::PrecacheVars: error loading vmt file for particle/warp_rain
CMaterial::PrecacheVars: error loading vmt file for particle/vistasmokev1_add_nearcull
CMaterial::PrecacheVars: error loading vmt file for particle/moth/moth
```
Might just be from the current map/addon rather than the previous map
